app/assets/javascripts/oxalis/model/reducers/settings_reducer.js

```diff
@@ -86,12 +86,14 @@ function SettingsReducer(state: OxalisState, action: Action): OxalisState {
         (result, layer) => {
           if (initialLayerSettings != null && initialLayerSettings[layer.name] != null) {
             result[layer.name] = initialLayerSettings[layer.name];
+            if (initialLayerSettings[layer.name].alpha == null) result[layer.name].alpha = 100;
```

Comment:
Can we change the logic here so that each initialLayerSetting is merged with the default settings (lines 93 to 96), so that only the missing values are set? That way, we wouldn't handle alpha as a special case (for which the default value is defined twice at the moment).